linux_mode/raw2dmp/raw2dmp.cc

```diff
@@ -0,0 +1,72 @@
+#include "../../src/libs/kdmp-parser/src/lib/kdmp-parser-structs.h"
```

Comment:
All right, I think it might be better to have this being done directly in the Python script that generate the memory dump. Based on the README this is currently done manually via a connection to the QEMU monitor, but is this something that could be automated once the breakpoint is hit?

This means it's transparent for the user, it also means we don't need to figure out CMakefiles / Makefile for this.